### ISSUE TITLE
Show certain parts of the mask as values are beeing entered possible?

### ISSUE THREAD

**Issue by globalser** [RobinHerbots/Inputmask#1870]
Hi Robin,

First of all, congratulations on your great job!
Is there anyway i can hide the underscores ( _ ) from the input mask and only show certain parts of the mask as digits are inserted?

This is what i have:

```
jQuery(document).ready(function($){
  $(".currency input").inputmask('€ 999,999.00', { numericInput: true });    //123456  =>  € ___.__1.234,56
});
```

Which displays:
€ ___.___,00

If i use:

```
jQuery(document).ready(function($){
  $(".currency input").inputmask('€ 999.999,00', { numericInput: true,placeholder:"€    .   ,00" });    //123456  =>  € ___.__1.234,56
});
```

Displays:
€ ,  ..  ,00

Althought on this last attempt, as i type the number appears correctly formatted.
The ideal case would be that for example, nothing shows up as mask. 
As the first digit is entered it would insert "€" before and ".00" after and the comma (",") only if more then 3 digits are inserted.

Is there anyway i could achieve this?

PS: I tried to add a fiddle, but i don't know why, i can't get the mask to show there.

**Comment by RobinHerbots** [comment]
@globalser ,

Try with 
```
 Inputmask("$ (,999){+|1}.99", {
                numericInput: true,
                placeholder: "0"
            }).mask(selector);
```

**Comment by globalser** [comment]
@RobinHerbots 

After adding:
$(".currency input").inputmask("€ (,999){+|1}.99", {numericInput: true,placeholder: "0"}).mask(selector);

The field acts like if there isn't any mask, as i type the numbers appear unformatted.

**Comment by RobinHerbots** [comment]
@globalser , 

Which version do you use of the inputmask?  Use version 4.0.0.0-beta.54

**Comment by globalser** [comment]
@RobinHerbots 

Sorry, I'm using wordpress and really not sure how to check the version i'm using.
This is the jquery version:


```
<script type='text/javascript' src='http://mydomain.com/wp-includes/js/jquery/jquery.js?ver=1.12.4'></script>
<script type='text/javascript' src='http://mydomain.com/wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1'></script>
```

I added: 

```
<script src="jquery.js"></script>
<script src="dist/jquery.inputmask.bundle.js"></script>
```

To header, without avail. Then to the footer, having the same problem (mask not showing).
Perhaps i'm doing something wrong.

Or it only works with the latest version of jquery?

**Comment by globalser** [comment]
Stupid me... i thought the src was an http and not local... will try downloading to the server.

**Comment by globalser** [comment]
I downloading and uploaded the code to wp-content folder.
After adding to the footer (the inputmask code comes after this line):

<script src="http://mydomain.com/wp-content/Inputmask-4.x/dist/jquery.inputmask.bundle.js"></script>

The problem stays the same. I can access the js file directly using the URL link, so it's not a typo.
And otherthing, do you happen to know why i can't create a fiddle for this issue?

https://jsfiddle.net/86fw3zug/ 
When i press run, it doesn't apply the mask. The first function has the 2 lines of the mask i'm working on, the second function was a copy paste from this one: https://jsfiddle.net/jonataswalker/2mLxkzw9/ 

If it's easier.. since the mentioned one @ https://jsfiddle.net/jonataswalker/2mLxkzw9/ is a workaround that suits my intentions, is there any way that the numbers after radixpoint (00) maintain as 00? So when i click, i can only edit the numbers on the left of the comma.

**Comment by RobinHerbots** [comment]
@globalser , 

https://jsfiddle.net/86fw3zug/2/

**Comment by globalser** [comment]
Thank you so much for your help Robin!

**Comment by RobinHerbots** [comment]
@globalser , 

Can this issue be closed?

**Comment by globalser** [comment]
@RobinHerbots 

Actually there is only more one small (i hope) thing.
The field is not limited in number of digits.

This is the final result i have https://jsfiddle.net/86fw3zug/6/ 
I want that the user is not able to insert more then 6 integer (after the "." they can't insert anything).

So 999,999.00 is correct, but 9,999,999.00 is incorrect.

As i've read, it's possible using input data-attribute, but that returns an error after submit. The ideal would be that it doesn't allow the user to insert more then 6 integer.
I added maxnumericlength based on this post https://github.com/RobinHerbots/Inputmask/issues/98 but without avail.

Tried adding each one of this options but don't do the job:
max:999999 
integerDigits:6
